gtsfm/scene_optimizer.py

```diff
@@ -321,14 +331,16 @@ def write_sfmdata_to_disk(sfm_data: SfmData, save_fpath: str) -> None:
 def aggregate_frontend_metrics(
     rot3_errors: List[Optional[float]],
     unit3_errors: List[Optional[float]],
+    corr_correctness: List[Tuple[int, float]],
     angular_err_threshold_deg: float,
 ) -> None:
     """Aggregate the front-end metrics to log summary statistics.
 
     Args:
-        rot3_errors: angular errors in rotations, measured in degrees
-        unit3_errors: angular errors in unit-translations, measured in degrees
-        angular_err_threshold_deg: threshold to classify the error as success, in degrees
+        rot3_errors: angular errors in rotations, measured in degrees.
+        unit3_errors: angular errors in unit-translations, measures in degrees.
+        corr_correctness: correctness of correspondences.
```

Comment:
ah yes. They will represent `(num_correct_correspondence, ratio_of_correct_correspondences)`

Reply:
judging from another file, it seems like 
```
	        Number of correct correspondences.
        Ratio of correspondences which are correct.
```
but I would not have guessed that from this docstring.

Is a NamedTuple more appropriate?
```python
class TwoViewCorrectnessResult(NamedTuple):
      num_correct_corr: float
      inlier_fraction: float
```

Reply:
Named tuple will create issues for dask

Reply:
we use it in data association without trouble though right?

Reply:
Yes we do. But I remember I had to change one of keypoints or images to a normal class because dask was unable to handle it. I guess it depends on whether the variable has to be passed around. In our case, it will be returned from a function

Reply:
Yes we do. But I remember I had to change one of keypoints or images to a normal class because dask was unable to handle it. I guess it depends on whether the variable has to be passed around. In our case, it will be returned from a function